prometheus-rules/prometheus-vmware-rules/alerts/datastore.alerts

```diff
@@ -341,3 +339,54 @@ groups:
     annotations:
       description: "Eph Datastore {{ $labels.datastore }} is not a part of correct SDRS Cluster. ({{ $labels.vcenter }}, {{ $labels.datacenter }})"
       summary: "Eph Datastore {{ $labels.datastore }} is not a part of correct SDRS Cluster. ({{ $labels.vcenter }}, {{ $labels.datacenter }})"
+
+  - alert: CinderTagMissingForDatastoreSSD
+    expr: >
+      vrops_datastore_cinder_tag{type="vmfs_p_ssd", cinder_tag="none"}
+      and on (datastore) vrops_datastore_cinder_state{cinder_state!="reserved"}
+    for: 5m
+    labels:
+      severity: info
+      tier: vmware
+      service: storage
+      support_group: compute
+      meta: "There is no Cinder tag present for the `{{ $labels.datastore }}` datastore in the *{{ $labels.storagepod }}* cluster\nLink to the vCenter: --> <{{ $labels.vcenter }}>"
+      playbook: docs/support/playbook/cinder/#
+      no_alert_on_absence: "true"
```

Comment:
@kevin-fischer @himanip94 
```suggestion
```
If you want to see absent alerts for the metrics you are using, this line can be omitted. See [the operator playbook for absent alerts](https://github.com/sapcc/absent-metrics-operator/blob/master/docs/playbook.md). 

This only applies to short-lived metrics such as the `*_alert_info` metric, which only exists when an alert event is captured by the vrops-exporter.

Reply:
Good to know.
I've made the changes and will be pushing them to another open PR with further additions.
Thx a lot Richard!